app/kubernetes-ingress-controller/kic-high-availability.md

```diff
@@ -0,0 +1,32 @@
+---
+title: KIC High Availability
+
+description: |
+  How to run multiple KIC instances with leader election
+content_type: reference
+layout: reference
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+---
+
+{{ site.kic_product_name }} reads state from the Kubernetes API server and generates a {{ site.base_gateway }} configuration. If {{ site.kic_product_name }} is not running, new {{ site.base_gateway }} instances will not receive a configuration. Existing {{ site.base_gateway }} instances will continue to process traffic using their existing configuration.
+
+Your {{ site.kic_product_name }} instance being offline is a major issue. The configuration loaded by {{ site.base_gateway }} will quickly become outdated, especially the upstream endpoints hosting your applications. Without {{ site.kic_product_name }} running, {{ site.base_gateway }} will not detect new application pods, or remove old pods from it's routing configuration.
+
+## Leader Election
+
+Kong recommends running at least _two_ {{ site.kic_product_name }} instances. {{site.kic_product_name}} elects a _leader_ when connected to a database-backed cluster or when Gateway Discovery is configured. This ensures that only a single controller pushes configuration to Kong's database or to Kong's Admin API to avoid potential conflicts and race conditions.
```

Comment:
```suggestion
Kong recommends running at least _two_ {{ site.kic_product_name }} instances. {{site.kic_product_name}} elects a _leader_ when connected to a database-backed cluster or when Gateway Discovery is configured. This ensures that only a single controller pushes configuration to {{site.base_gateway}}'s database or to {{site.base_gateway}}'s Admin API to avoid potential conflicts and race conditions.
```